modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/DateProcessorTests.java

```diff
@@ -105,14 +106,28 @@ public void testJodaPatternLocale() {
     }
 
     public void testJodaPatternDefaultYear() {
+        // fix joda's new DateTime() to Jan 12, 2017
+        DateTimeUtils.setCurrentMillisFixed(1484255848828L);
```

Comment:
this seems to be the way Joda set up to do such mock-hackery... I don't especially like it since this may affect other threads running during testing (not sure).